MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.m

```diff
@@ -552,4 +559,67 @@ - (void)selectScope:(id)sender
     [self.navigationController pushViewController:[MSALTestAppScopesViewController sharedController] animated:YES];
 }
 
+- (void)runStressTest:(id)sender
+{
+    (void)sender;
+    
+    void (^logHandler)(NSString *logLine) = ^(NSString *logLine) {
```

Comment:
Since I thought those test cases will end up being manually copied from the test case description and will not be part of the test app, I tried to have all the error logging in one method, but since it will be included in the test app (so the pain of copying will go away), I agree that it’s not a good solution :)!